### ISSUE TITLE
Can't install. Package 'uuidgen-0.3' is unavailable.

### ISSUE THREAD

**Issue by fakhrisatu** [ancane/markdown-preview-mode#32]
Hi, I am relatively new to Emacs.
When I tried execute `package-install RET markdown-preview-mode`, it gave me that error.
I followed the link you provided in README to `uuidgen`, but I didn't understand anything at all.
What exactly I need to do?

**Comment by KevinVanWorkum** [comment]
Having the same issue here. Emacs 24.3 on Centos-7.

**Comment by KevinVanWorkum** [comment]
I put this in my .emacs and installed the older of the 2 available (new one requires 24.4). I also had to install the ```discount``` package via ```yum install discount``` from epel on Centos.

```
(when (>= emacs-major-version 24)
  (require 'package)
  (add-to-list
   'package-archives
   '("melpa" . "http://melpa.org/packages/")
   '("melpa-stable" . "https://stable.melpa.org/packages/")
   )
  (package-initialize))

```

**Comment by Glutexo** [comment]
I had the same problem and even though I could go a step further to solve it, I had no luck either.

I downloaded the [uuidgen.el](https://github.com/kanru/uuidgen-el/blob/master/uuidgen.el) library from the linked repository, put it into Emacs’ _load-path_ and successfully loaded it. I can use the UUID functions, but _markdown-preview-mode_ still complains. To me it looks like that the _uuidgen_ should be registered as an installed package somewhere, but I didn’t find it in MELPA, neither I know how to do that manually. :(


I used this code to require the package:

```elisp
(add-to-list 'load-path "~/.emacs.d/custom")
(require 'uuidgen)
```
